docs/ingestion/ingestion-spec.md

```diff
@@ -507,26 +505,64 @@ is:
+With front coding enabled, Druid creates smaller UTF-8 encoded segments with very little performance cost.
+It involves tracking the length of common prefixes of values so that only the suffix is stored.
+Druid divides values into buckets each with a fixed number of values. Because the buckets are fixed, Druid can quickly determine which bucket to store a dictionary ID in, without hindering its ability to perform a binary search.
 
-:::info
- Front coding was originally introduced in Druid 25.0, and an improved 'version 1' was introduced in Druid 26.0, with typically faster read speed and smaller storage size. The current recommendation is to enable it in a staging environment and fully test your use case before using in production. By default, segments created with front coding enabled in Druid 26.0 are backwards compatible with Druid 25.0, but those created with Druid 26.0 or 25.0 are not compatible with Druid versions older than 25.0. If using front coding in Druid 25.0 and upgrading to Druid 26.0, the `formatVersion` defaults to `0` to keep writing out the older format to enable seamless downgrades to Druid 25.0, and then later is recommended to be changed to `1` once determined that rollback is not necessary.
+To enable front coding, set `indexSpec.stringDictionaryEncoding.type` to `frontCoded`.
+You can specify the following optional properties:
+* `bucketSize`: Number of values to place in a bucket to perform delta encoding. Setting this property instructs indexing tasks to write segments using compressed dictionaries of the specified bucket size. You can set it to any power of 2 less than or equal to 128. `bucketSize` defaults to 4.
+* `formatVersion`: Specifies older versions for backwards compatibility during rolling upgrades. Valid options are `0` and `1`. `formatVersion` defaults to `0`.
+
+For example:
+
+```
+"indexSpec": {
+  "stringDictionaryEncoding": {
+    “type”:”frontCoded”,
+    “bucketSize”: 4,
+    “formatVersion”:0
+  }
+}
+```
+
+A new `"formatVersion": 1` was introduced in Druid 26.0.0, offering typically faster read speeds and smaller storage sizes.
+When you upgrade from Druid 25.0.0 to Druid 26.0.0 while using front coding, the `formatVersion` defaults to `0` to keep writing out the older format. This default setting enables seamless downgrades to Druid 25.0.0 if needed.
+It is recommended to enable the new format in a staging environment and thoroughly test your use case before deploying it to production.
+
+:::caution
+By default, segments created with front coding enabled in Druid 26.0.0 are backwards compatible with Druid 25.0.0. Segments created with front coding in Druid 26.0.0 or 25.0.0 are not compatible with Druid versions older than 25.0.0.
 :::
 
-Beyond these properties, each ingestion method has its own specific tuning properties. See the documentation for each
```

Comment:
Did you remove this line intentionally? Could also be worth moving to intro, for example:

```
Some properties apply to all ingestion methods, but most are specific to a particular method.
See the documentation for each [ingestion method](./index.md#ingestion-methods) for details on the specific tuning properties for that method.

<newline>

The following example shows...
```

Reply:
I removed it because it seemed redundant. It links to the same source and the message is similar to 
`Some properties apply to all [ingestion methods](./index.md#ingestion-methods), but most are specific to each individual ingestion method.` Do you think I should add it back?